### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by imLEGR** [sd-webui/stable-diffusion-webui#1220]
### What happened?

When installing with the linux-sd.sh script, there is an error at the end.
![2022-09-19_06-15-54](https://user-images.githubusercontent.com/53097257/190944851-e3343b1f-63bc-4027-8978-8b85c4c4a9ff.png)

### Version

0.0.1 (Default)

### What browsers are you seeing the problem on?

Chrome

### Where are you running the webui?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by imLEGR** [comment]
ommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
I use fish

**Comment by JoshuaKimsey** [comment]
Yeah, my script is only designed to work with Bash. You'll need to use that to install and run it.
